packages/kbn-monaco/src/esql/lib/esql_theme.ts

```diff
@@ -169,23 +166,23 @@ export const buildESQLTheme = ({
           'param',
           'timespan_literal',
         ],
-        euiThemeVars.euiColorSuccessText
+        euiTheme.colors.textSuccess
       ),
     ],
     colors: {
-      'editor.foreground': euiThemeVars.euiTextColor,
-      'editor.background': euiThemeVars.euiColorEmptyShade,
-      'editor.lineHighlightBackground': euiThemeVars.euiColorLightestShade,
-      'editor.lineHighlightBorder': euiThemeVars.euiColorLightestShade,
-      'editor.selectionHighlightBackground': euiThemeVars.euiColorLightestShade,
-      'editor.selectionHighlightBorder': euiThemeVars.euiColorLightShade,
-      'editorSuggestWidget.background': euiThemeVars.euiColorEmptyShade,
-      'editorSuggestWidget.border': euiThemeVars.euiColorEmptyShade,
-      'editorSuggestWidget.focusHighlightForeground': euiThemeVars.euiColorEmptyShade,
-      'editorSuggestWidget.foreground': euiThemeVars.euiTextColor,
-      'editorSuggestWidget.highlightForeground': euiThemeVars.euiColorPrimary,
-      'editorSuggestWidget.selectedBackground': euiThemeVars.euiColorPrimary,
-      'editorSuggestWidget.selectedForeground': euiThemeVars.euiColorEmptyShade,
+      'editor.foreground': euiTheme.colors.textParagraph,
+      'editor.background': euiTheme.colors.backgroundBasePlain,
```

Comment:
It was `euiColorEmptyShade`, what color is the `backgroundBasePlain `?

Reply:
I imagine it's different but EUI advices that `emptyShade` be replaced with `backgroundBasePlain`, see https://github.com/elastic/kibana/issues/199715#issuecomment-2552310187, you'd have to scroll to the bottom.

Reply:
Ah I see, thanx Eyo!